R/zzz.R

```diff
@@ -75,14 +75,14 @@ fetch_GET <- function(x, store, path, args = NULL, format = NULL, key = NULL, ..
   }
   # set file format
   file_fmt <- file_fmt(x)
-  fmt <- ifelse(identical(file_fmt, character(0)), format, file_fmt)
+  fmt <- ifelse(is.na(file_fmt), format, file_fmt)
```

Comment:
same question about `is.na` here wrt can it lead to a vector of logicals > length 1